src/main/editDelete/preProcess.ts

```diff
@@ -2,13 +2,15 @@ import { Transcription, Word } from 'sharedTypes';
 
 const preProcessTranscript = (
   jsonTranscript: any,
-  duration: number
+  duration: number,
+  fileName: string
 ): Transcription => {
   const numberOfWords: number = jsonTranscript.words.length;
 
   const processedTranscript: Transcription = {
     confidence: jsonTranscript.confidence,
     words: [],
+    fileName,
```

Comment:
Everything's looking good!
Was just wondering: Should we attach the filename to a Transcription or a Word object? The Word interface currently has a "filename" field which is unused. I'm unsure of the correct answer, and it would probably matter when we start to have multiple videos layered on top of one another.

Reply:
Hmm yea I did see that there's a filename field for each word. I'm open to storing it however it's gonna be easiest. I'm currently not sure what the filename is gonna be used for, something like  identifying which words come from which files? Would it be better to store the full file path for that then?

Reply:
Not sure either. I suppose it shouldnt matter right now since its a simple change to make and we don't have a clear picture of how this will be used right now.

Reply:
> Everything's looking good! Was just wondering: Should we attach the filename to a Transcription or a Word object? 

Down the line we might have multiple audio sources. So to make future development easier I think it should be attached to both.

In the case that we end up developing a version with multiple audio source and the `fileName` attribute it unused, then it could be removed.

Reply:
I am fine with using media path as an identifier of sorts right now, but down the line we should have a unique identifier that doesn't change in case the media file gets moved

Reply:
With regards to where it's used, Word, Transcription, either is fine - it's easily changeable later

Reply:
Yea, very easy change if we need it as a path/name and whether it's stored in words or transcriptions. For now, I've pushed a commit to save it in words and not transcriptions, just so we don't have differing ways of accessing the same value if it's not functionally required.